recipe/meta.yaml

```diff
@@ -1,28 +1,29 @@
 {% set name = "rhumba" %}
-{% set version = "0.0.2" %}
+{% set version = "0.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/mamba-org/rhumba/archive/{{ version }}.tar.gz
-  sha256: ebe644cce7bc88fdee7e0c632c15003ca7fbd6dafb663a5062b763b5db3f320a
+  sha256: 4c863e762b0936673694fab99a805ec7b1fcd5954721d659de807e41193ff16e
 
 build:
+  merge_build_host: true # [win]
   number: 0
   rpaths:
     - lib/R/lib/
     - lib/
-  skip: true  # [win]
 
 requirements:
   build:
     - {{ compiler('c') }}              # [not win]
     - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('r_clang') }} # [win]
```

Comment:
woooot this is possible? We could have used C++17 all along?! :)

Reply:
just tried some cargo culting here, not working!